transmart-api-server/grails-app/conf/logback.groovy

```diff
@@ -37,3 +40,47 @@ if (Environment.isDevelopmentMode() && targetDir != null) {
 else {
     root(INFO, ['STDOUT'])
 }
+
+boolean productionMode = Environment.current == Environment.PRODUCTION
+def logDirectory = BuildSettings.TARGET_DIR
+if (productionMode) {
+    def catalinaBase = System.getProperty('catalina.base') ?: '.'
+    logDirectory = "${catalinaBase}/logs".toString()
+}
+
+/**
+ * Configuration for writing audit metrics.
+ * This could be placed in the out-of-tree Config.groovy and will override the configuration below.
+ * See https://logback.qos.ch/manual/appenders.html for details on configuration
+ */
+appender('fileAuditLogger', RollingFileAppender) {
+    file = "${logDirectory}/audit.log"
+    rollingPolicy(SizeAndTimeBasedRollingPolicy) {
+        // daily rollover
+        fileNamePattern = "${logDirectory}/audit.%d{yyyy-MM-dd}.%i.log"
+        // size limit on the log files
+        maxFileSize = '100MB'
+        /* // optional parameters
+        // maxHistory controls the maximum number of archive files to keep, asynchronously deleting older files.
+        maxHistory = 30
+        // totalSizeCap controls the total size of all archive files.
+        // Oldest archives are deleted asynchronously when the total size cap is exceeded
+        totalSizeCap = '3GB'
```

Comment:
this parameters are also useful, but are optional, so I commented them out for now. Please let me know, if it is better to have them there by default.